.github/workflows/reusable-container-workflow.yaml

```diff
@@ -132,7 +132,7 @@ jobs:
       #  build is broken based on platforms as load: true is not supported with multi-platform builds
       - if: ${{ hashFiles(format('{0}-{1}', matrix.dockerfile, inputs.build_type)) }}
         name: Build release image for amd64
-        uses: docker/build-push-action@v3
```

Comment:
this jumped from v3 to v5. is this intended?

Reply:
yes, see an example here:
https://github.com/docker/build-push-action